python/ray/dag/compiled_dag_node.py

```diff
@@ -681,13 +684,10 @@ def __init__(
         # Mapping from the actor handle to the node ID that the actor is on.
         self.actor_to_node_id: Dict["ray.actor.ActorHandle", str] = {}
 
-        # This is set to true when type hint of `transport="nccl"` is used.
-        self._use_default_nccl_group = False
-        # This is set to the specified custom nccl group
-        # if there exists a type hint of `transport=nccl_group`.
-        self._custom_nccl_group_p2p: Optional[GPUCommunicator] = None
-        # The NCCL group ID for P2P send/recv operations.
-        self._nccl_group_id_p2p: Optional[str] = None
+        # This is set to the custom nccl group specified in `experimenatal_compile`.
```

Comment:
Nit: `custom_nccl_group`.

Reply:
Nit: The `custom_nccl_group` specified in `experimenatal_compile`.

Reply:
Done.